tests/SmokeTests/CrawlerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SmokeTests;
+
+use App\Tests\SmokeTests\ProjectWebTestCase;
+
+final class CrawlerTest extends ProjectWebTestCase
+{
+    public function testFrontpageLinks(): void
+    {
+        $client = parent::createClient();
+        $crawler = $client->request('GET', '/');
+
+        $this->assertResponseIsSuccessful();
+
+        $frontpageLinks = $crawler->filter('a')->links();
+
+        $selectedLinkIndices = array_rand($frontpageLinks, 10);
+
+        foreach($selectedLinkIndices as $index) {
+            $link = $frontpageLinks[$index];
+            fwrite(STDERR, print_r($link->getUri() . PHP_EOL, TRUE));
```

Comment:
Can we use Symfony console here? Or see if PHPUnit has it's own output writer?

Reply:
Does not seem that PHPUnit has anything, but changed to symfony output in 25ea341f